uploader/main.js

```diff
@@ -0,0 +1,21 @@
+"use strict";
+
+const version = process.env.NW_VERSION;
+
+const download = require("./download");
+const upload = require("./upload");
+const packageUpdater = require("./package-updater");
+
+const downloads = download(version);
```

Comment:
Here's the bit where it looks weird. Wouldn't it be better to have `downloads = downloadAll(version);`?

Reply:
It doesn't look weird to me. After all the whole point of the module is to download all the versions of NW.js to extract the chromedriver and upload to S3. So the "all" seems superfluous.